vuepress/docs/docs/schema.md

```diff
@@ -292,6 +292,9 @@ You can also indicate that the effect type is abstract via `--abstractEffectType
 If you want to force a mapping between a GraphQL type and a Scala class (such as scalars), you can use the
 `--scalarMappings` option. Also you can add additional imports by providing `--imports` option.
 
+Since Caliban 1.2.4, you can generate schemas using an sbt `sourceGenerator`, which means your schemas will be generated every time you compile (or when you import your build into [Metals](https://scalameta.org/metals/)).
```

Comment:
This would probably be better written as a normal section like in the Client docs, but I didn't want to spend much time working on these docs for now.

Reply:
It will be from 1.3.0 (I don't think it was part of 1.2.4).

Reply:
true, I'll update that